repo/remote.go

```diff
@@ -25,12 +25,12 @@ func RemoteAdd(name, url string) *exec.Cmd {
 	return cmd
 }
 
-func RemotePush(remote, branch string, force bool) *exec.Cmd {
+func RemotePush(remote, localbranch string, branch string, force bool) *exec.Cmd {
```

Comment:
Fixed by splitting into RemotePushNamedBranch, and refactoring RemotePush to call that function with a hardcoded parameter.